$MirOS: ports/www/dillo/v2/patches/patch-dpi_downloads_cc,v 1.1 2008/11/07 18:05:52 bsiegert Exp $
--- dpi/downloads.cc.orig	Sun Jun  7 05:40:25 2009
+++ dpi/downloads.cc	Sun Aug 23 17:03:44 2009
@@ -333,7 +333,7 @@ DLItem::DLItem(const char *full_filename
       dl_argv[i++] = (char*)"-c";
    }
    dl_argv[i++] = (char*)"--load-cookies";
-   dl_argv[i++] = dStrconcat(dGethomedir(), "/.dillo/cookies.txt", NULL);
+   dl_argv[i++] = dStrconcat(dGethomedir(), "/.etc/dillo/cookies.txt", NULL);
    dl_argv[i++] = (char*)"-O";
    dl_argv[i++] = fullname;
    dl_argv[i++] = esc_url;
